frontend/.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_API="http://localhost:3001"
\ No newline at end of file
+REACT_APP_BACKEND_API="http://localhost:3010"
```

Comment:
This file should likely be ignored and removed from git, to stop accidentally leaking secrets.

We can .gitignore it, and it's also good practice to create an `example.env` file which has all the fields (but not values) to make first time setup easier.

Reply:
We can discuss on it and for now we will not check-in the secrets in .env for dev as well